eva-server/src/main/resources/application.properties

```diff
@@ -19,4 +19,13 @@ springfox.documentation.swagger.v2.path=/webservices/api
 #spring.profiles.active=oauth2-security
 #security.oauth2.resource.user-info-uri = ...
 
-eva.mongo.collections.files=@eva.mongo.collections.files@
+spring.data.mongodb.host=@eva.mongo.host@
+spring.data.mongodb.authentication-database=@eva.mongo.auth.db@
+spring.data.mongodb.username=@eva.mongo.user@
+spring.data.mongodb.password=@eva.mongo.passwd@
+spring.data.mongodb.read-preference=@eva.mongo.read-preference@
+
+eva.collections.variants=@eva.mongo.collections.variants@
+eva.collections.files=@eva.mongo.collections.files@
```

Comment:
These properties are defined differently in the pipeline: those properties are still defined in the pipeline: https://github.com/EBIvariation/eva-pipeline/blob/develop/examples/load-genotyped-vcf.properties#L40-L41